Unhandled exception
Traceback (most recent call last):
  File "./app/tasks.py", line 237, in set_images_lazy
    art = Article.query.get(a.id)
  File "/Users/rr/Desktop/RR/code/LearnBetter/venv/lib/python3.6/site-packages/sqlalchemy/orm/attributes.py", line 287, in __get__
    return self.impl.get(instance_state(instance), dict_)
  File "/Users/rr/Desktop/RR/code/LearnBetter/venv/lib/python3.6/site-packages/sqlalchemy/orm/attributes.py", line 718, in get
    value = state._load_expired(state, passive)
  File "/Users/rr/Desktop/RR/code/LearnBetter/venv/lib/python3.6/site-packages/sqlalchemy/orm/state.py", line 652, in _load_expired
    self.manager.deferred_scalar_loader(self, toload)
  File "/Users/rr/Desktop/RR/code/LearnBetter/venv/lib/python3.6/site-packages/sqlalchemy/orm/loading.py", line 946, in load_scalar_attributes
    "attribute refresh operation cannot proceed" % (state_str(state))
sqlalchemy.orm.exc.DetachedInstanceError: Instance <Article at 0x104cc9da0> is not bound to a Session; attribute refresh operation cannot proceed (Background on this error at: http://sqlalche.me/e/bhk3)
